plugins/nginx-vhosts/install

```diff
@@ -4,10 +4,12 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 case "$DOKKU_DISTRO" in
   ubuntu)
     echo "%dokku ALL=(ALL) NOPASSWD:/etc/init.d/nginx reload, /usr/sbin/nginx -t" > /etc/sudoers.d/dokku-nginx
+    echo "%dokku ALL=(ALL) NOPASSWD:/usr/bin/tail" >> /etc/sudoers.d/dokku-nginx
     ;;
 
   opensuse)
     echo "%dokku ALL=(ALL) NOPASSWD:/sbin/service nginx reload, /usr/sbin/nginx -t" > /etc/sudoers.d/dokku-nginx
+    echo "%dokku ALL=(ALL) NOPASSWD:/usr/bin/tail" >> /etc/sudoers.d/dokku-nginx
```

Comment:
I think this can be pared down to `/usr/bin/tail /var/log/nginx/*` and `/usr/bin/tail -f /var/log/nginx/*`. Test this of course. :smile: